infra/base-images/base-builder/compile

```diff
@@ -186,6 +200,25 @@ else
   fi
 fi
 
+if [ "$SANITIZER" = "introspector" ]; then
+  unset CXXFLAGS
+  unset CFLAGS
+  apt-get install -y libjpeg-dev zlib1g-dev
```

Comment:
FYI installing packages like this can mess up compilation of some projects I think